docs/DXIL.rst

```diff
@@ -515,33 +515,52 @@ Additional shader properties are specified via tag-value pair list, which is the
+0        Disable shader optimizations                                                             Command-line flag ``-disable-llvm-optzns`` provided to DXC's internal frontend
+1        Disable math refactoring                                                                 Not set
+2        Double-precision floating point                                                          Use of the double data type
+3        Force early depth-stencil test                                                           Shader is a pixel shader, and the ``[earlydepthstencil]`` attribute is present in the HLSL source
+4        Raw and structured buffers                                                               Use of RawBuffer or StructuredBuffer resource types
+5        Low-precision data types                                                                 Use of half or i16 data types
+6        Double-precision extensions                                                              Use of FDiv, UIToFP, SIToFP, FPToUI, FPToSI, or Fma instructions with a double type
+7        Masked sum of absolute differences                                                       Use of the Msad instruction
+8        All resources bound for the duration of shader execution                                 Command-line flag ``/all_resources_bound`` provided to DXC
+9        View port or render target array index from any shader feeding rasterizer                ViewPortArrayIndex or RenderTargetArrayIndex semantics are present in the input signature of a geometry shader, or the output signature of a vertex, domain, or hull shader. If DXIL validator version < 1.4, this flag is also required if the ViewPortArrayIndex or RenderTargetArrayIndex semantics are present in the input signature of domain and hull shaders
+10       Inner coverage                                                                           Use of the InnerCoverage instruction, or the InnerCoverage semantic is present in the output signature of a pixel shader
+11       Stencil reference value                                                                  The StencilRef semantic is present in the output signature of a pixel shader
+12       Tiled resources                                                                          Use of the CheckAccessFullyMapped instruction or, if DXIL validator version >= 1.8, the use of LodClamp in intrinsics SampleGrad, SampleCmpGrad, Sample, SampleBias, SampleCmp, or SampleCmpBias
+13       Typed UAV load additional formats                                                        Use of TextureLoad or BufferLoad on a UAV with a multi-component data type. If DXIL validator version == 1.0 then this flag is set when a TextureLoad or BufferLoad is used on any UAV, regardless of its data type
+14       Comparison filtering for feature level 9                                                 Not set
```

Comment:
It would be good to know the context in which this was valid, but I suspect this was also removed in SM 6.0.